cart-service/src/lib/dovetech-response-mapper.ts

```diff
@@ -268,15 +262,18 @@ const buildSetCustomTypeAction = (
     fields[COMMIT_ID] = commitId;
   }
 
-  const setCustomTypeAction: CartSetCustomTypeAction = {
+  const setCustomTypeMetadataAction: CartSetCustomTypeAction = {
     action: 'setCustomType',
     type: {
-      key: CART_METADATA,
+      key: EXTENSION_TYPES_DATA_KEY,
       typeId: 'type',
     },
     fields: fields,
   };
-  return setCustomTypeAction;
+
+  const actions: CartUpdateAction[] = [setCustomTypeMetadataAction];
+
+  return actions;
```

Comment:
This function can stay as just returning one action rather than an array now

Reply:
just thought that would leave as an array until we know why they do not let more than type per object at a time. Thought they might give us a reason, or a way to actually use them